integration-tests/bats/sql-patch.bats

```diff
@@ -0,0 +1,391 @@
+#!/usr/bin/env bats
+load $BATS_TEST_DIRNAME/helper/common.bash
+
+setup() {
+    setup_common
+}
+
+teardown() {
+    assert_feature_version
+    teardown_common
+}
+
+@test "sql-patch: output reconciles INSERT query" {
```

Comment:
I didn't notice a test for the `--cached` flag. I wanted to make sure we were executing it correctly since the doc string was switched around before.